include/mcl_3dl/raycasts/raycast_using_dda.h

```diff
@@ -77,6 +77,12 @@ class RaycastUsingDDA : public Raycast<POINT_TYPE>
   {
     kdtree_ = kdtree;
     updatePointCloud();
+    if (!isPointWithinMap(ray_begin))
+    {
+      max_movement_ = 0;
+      pos_ = 0;
+      return;
```

Comment:
I think raycaster still should cast the ray by handling out-of-the-map as no-hit.
Otherwise, it causes a problem at the open end of the map.

(For very temporary fix, it is acceptable though.)

Reply:
Merging for now.
https://github.com/at-wat/mcl_3dl/issues/352